src/components/Card/Card.tsx

```diff
@@ -1,23 +1,87 @@
-import './Card.scss'
+import { useState } from "react";
+import "./Card.scss";
+
+interface IPropsData {
+  title: string;
+  description: string;
+  id: string;
+}
 
 interface IProps {
-    title: string;
-    description: string;
-    id: string;
+  data: IPropsData[];
+  removeCardHandler: (id: string) => void;
+  editCardHandler: () => void;
+  isEdit: boolean;
 }
 
-const Card = (props: { data: IProps[]; }) => {
-const {data} = props
+const Card = (props: IProps) => {
+    const { data, removeCardHandler, editCardHandler, isEdit } =
+        props;
+
+    const [enteredTitle, setEnteredTitle] = useState("");
+    const [enteredText, setEnteredText] = useState("");
+
+    const titleChangeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+        let item: IPropsData = data.find(el=>el.id===event.target.id)!
+
+        setEnteredTitle(event.target.value);
+        item.title = enteredTitle
+    };
+
+    const textChangeHandler = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+        let item: IPropsData = data.find(el=>el.id===event.target.id)!
+
+        setEnteredText(event.target.value);
+        item.description = enteredText
+    };
+
+    const saveCardHandler = (event: { preventDefault: () => void; }) => {
+        event.preventDefault()
+        editCardHandler();
+    };
 
     return (
+        <div className="content-container">
+        <div >
+            <button className="btn-add" onClick={editCardHandler}>
+            Edit
```

Comment:
Here should be btn 'Cancel' in the edit mode. It should clear all the done changes.

Reply:
done